.github/ISSUE_TEMPLATE.md

```diff
@@ -20,6 +20,7 @@ Description block and provide responses for **all** of the below items.
 * Docker image used
 * Operating System
 * `docker info` output
+* `docker inspect $CTID` output
```

Comment:
I stopped here for a few seconds. May be instead of `$CTID` it'd be clearer is we say `<container_id> or <container_name>`?

Reply:
ugh, my openvz is sowing. Dockerhub shows the docs as `NAME|ID` for the `docker inspect` docs. I can do your suggestion or that, it matters not to me. You pick :D